flink-streaming-java/src/main/java/org/apache/flink/streaming/api/environment/CheckpointConfig.java

```diff
@@ -285,6 +288,24 @@ public boolean isExternalizedCheckpointsEnabled() {
 		return externalizedCheckpointCleanup != null;
 	}
 
+	/**
+	 * Returns whether a job recovery should fallback to checkpoint when there is a savepoint later than checkpoint.
```

Comment:
I would replace in all the docs : " when there is a savepoint later than checkpoint" with "when there is a more recent savepoint"